### PR TITLE
Fix overflow into TOC on small screens

### PR DESCRIPTION
**Pull request by kirbyk**

### CODE REVIEW

File: themes/mongodb/src/css/mongodb-docs.css [hunk 1]
```diff
@@ -704,7 +704,7 @@ div.versionadded > p > span, div.versionchanged > p > span, div.deprecated > p >
 .content .main-column {
     order: 1;
     min-height: 600px;
-    width: 85%;
+    width: 68%;
```
Comment (i80and) [inline themes/mongodb/src/css/mongodb-docs.css]: Is there a reason for this constant, or is it just "what works"?

**Comment by i80and** [comment]
Merged in 7fa9fef62a68444427e888ff26140de5fb7d986f

File: themes/mongodb/src/css/mongodb-docs.css (hunk 1 above)
Comment (kirbyk) [inline themes/mongodb/src/css/mongodb-docs.css]: It's a mix of both. Any percentage under 69% will work.

The TOC is fixed at 330px, and screen widths over 1093px display the TOC.

1 - 330/1093 = 69.8%.

**Comment by kirbyk** [comment]
Oh, sorry! I almost asked about that. It gets called when we deploy though, no?

**Pull request closed without merging**